### PR TITLE
Add openapi validation schema to CRDs

### PR DESCRIPTION
**Pull request by omeryahud**
Signed-off-by: Omer Yahud <oyahud@redhat.com>

### CODE REVIEW

**Comment by ksimon1** [comment]
@tiraboschi can you please review this PR?

File: cluster/latest/kubevirt-ssp-operator-crd.yaml [hunk 1]
```diff
@@ -65,4 +115,22 @@ spec:
...
+          type: object
+        spec:
+          type: object
+          properties:
+            version:
+              type: string
+            templateValidatorReplicas:
+              type: integer
+        status:
+          type: object
```
Comment (tiraboschi) [inline cluster/latest/kubevirt-ssp-operator-crd.yaml:136]: a newline here

File: deploy/crds/kubevirt_v1_templatevalidator_crd.yaml [hunk 2]
```diff
@@ -13,4 +13,22 @@ spec:
...
+          type: object
+        spec:
+          type: object
+          properties:
+            version:
+              type: string
+            templateValidatorReplicas:
+              type: integer
+        status:
+          type: object
```
Comment (tiraboschi) [inline deploy/crds/kubevirt_v1_templatevalidator_crd.yaml:34]: same

File: manifests/generated/kubevirt-ssp-operator.vVERSION.clusterserviceversion.yaml [hunk 3]
```diff
@@ -1,12 +1,9 @@
...
-    capabilities: Basic Install
-    categories: Openshift Optional
-    containerImage: REPLACE_IMAGE
-    description: Manages KubeVirt addons for Scheduling, Scale, Performance
+  annotations: {alm-examples: '[{"apiVersion":"ssp.kubevirt.io/v1","kind":"KubevirtCommonTemplatesBundle","metadata":{"name":"kubevirt-common-template-bundle"},"spec":{"version":"v0.7.0"}},{"apiVersion":"ssp.kubevirt.io/v1","kind":"KubevirtMetricsAggregation","metadata":{"name":"kubevirt-metrics-aggregation"},"spec":{"version":"v0.0.1"}},{"apiVersion":"ssp.kubevirt.io/v1","kind":"KubevirtNodeLabellerBundle","metadata":{"name":"kubevirt-node-labeller-bundle"},"spec":{"version":"v0.1.1"}},{"apiVersion":"ssp.kubevirt.io/v1","kind":"KubevirtTemplateValidator","metadata":{"name":"kubevirt-template-validator","namespace":"kubevirt"},"spec":{"templateValidatorReplicas":2,"version":"v0.6.2"}}]',
+    capabilities: Basic Install, categories: Openshift Optional, containerImage: REPLACE_IMAGE,
```
Comment (tiraboschi) [inline manifests/generated/kubevirt-ssp-operator.vVERSION.clusterserviceversion.yaml:5]: something got broken here collapsing 4 lines

File: manifests/kubevirt-ssp-operator/latest/kubevirt-ssp-operator.latest.clusterserviceversion.yaml [hunk 4]
```diff
@@ -2,12 +2,9 @@
...
-    capabilities: Basic Install
-    categories: Openshift Optional
-    containerImage: REPLACE_IMAGE
-    description: Manages KubeVirt addons for Scheduling, Scale, Performance
+  annotations: {alm-examples: '[{"apiVersion":"ssp.kubevirt.io/v1","kind":"KubevirtCommonTemplatesBundle","metadata":{"name":"kubevirt-common-template-bundle"},"spec":{"version":"v0.7.0"}},{"apiVersion":"ssp.kubevirt.io/v1","kind":"KubevirtMetricsAggregation","metadata":{"name":"kubevirt-metrics-aggregation"},"spec":{"version":"v0.0.1"}},{"apiVersion":"ssp.kubevirt.io/v1","kind":"KubevirtNodeLabellerBundle","metadata":{"name":"kubevirt-node-labeller-bundle"},"spec":{"version":"v0.1.1"}},{"apiVersion":"ssp.kubevirt.io/v1","kind":"KubevirtTemplateValidator","metadata":{"name":"kubevirt-template-validator","namespace":"kubevirt"},"spec":{"templateValidatorReplicas":2,"version":"v0.6.2"}}]',
+    capabilities: Basic Install, categories: Openshift Optional, containerImage: REPLACE_IMAGE,
```
Comment (tiraboschi) [inline manifests/kubevirt-ssp-operator/latest/kubevirt-ssp-operator.latest.clusterserviceversion.yaml:6]: same

File: manifests/kubevirt-ssp-operator/latest/kubevirt_v1_templatevalidator_crd.yaml [hunk 5]
```diff
@@ -13,4 +13,22 @@ spec:
...
+          type: object
+        spec:
+          type: object
+          properties:
+            version:
+              type: string
+            templateValidatorReplicas:
+              type: integer
+        status:
+          type: object
```
Comment (tiraboschi) [inline manifests/kubevirt-ssp-operator/latest/kubevirt_v1_templatevalidator_crd.yaml:34]: same

**Comment by omeryahud** [comment]
Will follow up with a new PR, probably will bump operator-sdk version

**Pull request closed without merging**